apps/avifenc.c

```diff
@@ -421,8 +484,9 @@ int main(int argc, char * argv[])
     int maxQuantizer = 26;
     int minQuantizerAlpha = AVIF_QUANTIZER_LOSSLESS;
     int maxQuantizerAlpha = AVIF_QUANTIZER_LOSSLESS;
-    int tileRowsLog2 = 0;
-    int tileColsLog2 = 0;
+    int tileRowsLog2 = -1;
+    int tileColsLog2 = -1;
```

Comment:
Optional: `avifBool manualTiling = AVIF_FALSE;` might be simpler and no need for the `tileRowsLog2 < 0` checks below.

Reply:
Thanks for the suggestion. This approach has the awkward property that `manualTiling` is not always the opposite of `autoTiling`, because `manualTiling` actually means whether `--tilerowslog2` or `--tilecolslog2` is specified. We can address this issue by renaming this boolean flag, such as `tilingExplicitlySet` or `tileConfigurationExplicitSet`, similar to the `cicpExplicitlySet` boolean flag.